### PR TITLE
Update font colours and weight

### PR DESCRIPTION
**Pull request by kopijunkie**
# Task
* Update font colours and font weights according to new specs following DAC Audit
* Applies to all screens on JS SDK

# New Specs (from CX-4587)
**TITLES (e.g. 'Verify your identity')**
color: #2C3E4F; --> _change to #2b2d33_
font-weight: 600;

**SUB-TITLES (e.g. 'Select the type of document you would like to upload')**
color: #636670; --> _change to #1e1e24_
font-weight: 600;

**DOC SELECTOR BUTTON TITLE (e.g. 'Passport')**
color: #2C3E4F; --> _change to #1e1e24_
font-weight: 600;

**DOC SELECTOR BUTTON DESCRIPTION (e.g. 'Face photo page')**
color: #565B5E; --> _change to #1e1e24_
font-weight: 400; --> _change to 500_

**MAIN BODY COPY**
color: #636670; --> _change to #1e1e24_
font-weight: 400; --> _change to 500_

**BUTTON TEXT**
All button text should also have font weight changed to 500 

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [ ] Has the README been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-913-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://913-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-913-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-913-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://913-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-913-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-913-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-913-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-913-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://913-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-913-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-913-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://913-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-913-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-913-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://913-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-913-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-913-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://913-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Theme/constants.css [hunk 1]
```diff
@@ -59,21 +59,21 @@
...
 @color-border: @color-neutral-600;
 @color-divider: #DEE1E4;
 @color-spinner: @color-neutral-600;
-@color-body-text: @color-primary-700;
+@color-body-text: #1E1E24;
+@color-title-text: #2B2D33;
+@color-subtitle-text: @color-body-text;
 
-@color-text-dark: #2C3E4F;
+@color-text-dark: #1E1E24;
```
Comment (stefaniacardenas) [inline src/components/Theme/constants.css]: `@color-body-text` and `@color-text-dark` have the same value. When do we use `@color-text-dark`, and why can we not use `@color-body-text directly?

Comment (kopijunkie) [reply src/components/Theme/constants.css]: Good point. `@color-text-dark` seems to be used across a few components' stylesheets (`CrossDeviceLink`, `PhoneNumberInput`, `NavigationBar`, `PoAIntro` to name a few) but they really are the same thing now. Will update.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-913-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-913-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://913-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-913-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-913-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://913-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**